src/NServiceBus.Core/obsoletes.cs

```diff
@@ -457,7 +468,8 @@ public SendOptions ToSendOptions(string replyToAddress)
         [ObsoleteEx(
             Message = "Not used anymore",
             RemoveInVersion = "7.0",
-            TreatAsErrorFromVersion = "6.0")] public const string OriginalReplyToAddress = "NServiceBus.Timeout.ReplyToAddress";
+            TreatAsErrorFromVersion = "6.0")]
```

Comment:
Did you do a reformat on this file? Cause these two changes are not relevant to this PR